### DISCUSSION TITLE
Missing element: expected ".tabbed-labels" to be present

### DISCUSSION THREAD

**Discussion by berg223** [squidfunk/mkdocs-material#5703] [Q&A] [answered]
Some specific [article](https://xiaocairush.github.io/ds/stack/) report the following error in console:

```
index.ts:77 Uncaught ReferenceError: Missing element: expected ".tabbed-labels" to be present
    at z (index.ts:77:11)
    at Fo (index.ts:136:21)
    at index.ts:114:21
    at Array.map (<anonymous>)
    at Uo (index.ts:114:8)
    at bundle.ts:251:16
    at Array.map (<anonymous>)
    at bundle.ts:251:6
    at e._subscribe (defer.ts:55:15)
    at e._trySubscribe (Observable.ts:244:19)
```

However, article like [this](https://xiaocairush.github.io/ds/queue/) won't report error. Is this a bug? How to fix it?

Here is my [repo of source code](https://github.com/xiaocairush/xiaocairush.github.io).

**Comment by squidfunk** [comment, marked as answer]
Did you configure content tabs correctly? It's likely related to the missing [`alternate_style`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#+pymdownx.tabbed.alternate_style) setting:

``` yaml
markdown_extensions:
  - pymdownx.superfences
  - pymdownx.tabbed:
      alternate_style: true 
```

**Comment by berg223** [comment]
Thanks @squidfunk It worked! You are my hero!
